packages/react/src/components/ValidationMessage/ValidationMessage.tsx

```diff
@@ -8,8 +8,11 @@ import type { MergeRight } from '../../utilities';
 export type ValidationMessageProps = MergeRight<
   Omit<DefaultProps, 'data-color'> & HTMLAttributes<HTMLParagraphElement>,
   {
-    /** Toggle error color */
-    error?: boolean;
+    /**
+     * Sets color and icon.
+     * @default 'danger'
+     */
+    'data-color'?: 'danger' | 'success';
```

Comment:
Maybe pick these colors from `SeverityColors` from `src/colors.ts`? So we make sure to use the same type for all severity colors :)

Reply:
I intentionally did not use those colors, as design have not given green light to add all the colors to ValidationMessage as mentioned in https://designsystemet.slack.com/archives/C07K7NEKXEW/p1733217765654359 :) Do you want to wait for design, or should we merge this first?

Reply:
I meant we could just `Pick<>` `success` and `danger` from that type.

Reply:
I didn't mean to use all the colors just "pick", `danger` and `success` from `SeverityColors`. Sorry for the confusing, its the [Extract](https://www.typescriptlang.org/docs/handbook/utility-types.html#extracttype-union) utility type, so `Extract<SeverityColors, 'success' | 'danger'>`

Reply:
Ah! Good point :)